README.md

```diff
@@ -32,30 +51,29 @@ services:
     image: "wemakeservices/caddy-gen:latest"
     restart: always
     volumes:
-      - /var/run/docker.sock:/tmp/docker.sock:ro  # needs socket to read events
-      - ./certs/acme:/etc/caddy/acme  # to save acme
-      - ./certs/ocsp:/etc/caddy/ocsp  # to save certificates
+      - /var/run/docker.sock:/tmp/docker.sock:ro # needs socket to read events
+      - ./certs/acme:/etc/caddy/acme # to save acme
+      - ./certs/ocsp:/etc/caddy/ocsp # to save certificates
```

Comment:
I though that some paths have changed, haven't they?

Reply:
:heavy_check_mark: